File: documentation/user/en/query/requirements/examples/fetching/dataInLocales.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Brand"),
				filterBy(
					entityPrimaryKeyInSet(64703)
				),
				require(
					entityFetch(
						attributeContent("code", "name"),
						dataInLocales(Locale.forLanguageTag("cs"))
					)
				)
			)
		);
	}
);

File: documentation/user/en/query/requirements/examples/fetching/dataInLocales.cs
EvitaResponse<ISealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Brand"),
        	FilterBy(
        		EntityPrimaryKeyInSet(64703)
        	),
        	Require(
        		EntityFetch(
        			AttributeContent("code", "name"),
        			DataInLocales(CultureInfo.GetCultureInfo("cs"))
        		)
        	)
        )
	)
);